modules/virt-querying-metrics.adoc

```diff
@@ -1,10 +1,10 @@
 // Module included in the following assemblies:
-//
-// * virt/support/virt-prometheus-queries.adoc
+    //
```

Comment:
There seem to be extra leading spaces on lines 2,3,5,6, and 7. Can you please correct the spacing?